io.github.electron.geforcenow.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.electron.geforcenow</id>
+  <launchable type="desktop-id">geforcenow-electron.desktop</launchable>
+  <name>GeForce Now Electron</name>
+  <project_license>CC0-1.0</project_license>
+  <developer_name>hmlendea</developer_name>
+  <summary>Cloud gaming electron app</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://github.com/hmlendea/geforcenow-electron</url>
+  <url type="help">https://github.com/hmlendea/geforcenow-electron/issues</url>
+  <url type="bugtracker">https://github.com/hmlendea/geforcenow-electron/issues</url>
+  <description>
+    <p>GeForce Now application written in Electron, wrapping around the GFN web browser implementation.</p>
```

Comment:
I think it should be stated it's not an official app and it's not affiliated with Nvidia. See DataGrip for an example: https://flathub.org/apps/details/com.jetbrains.DataGrip

Reply:
I've changed the description